### DISCUSSION TITLE
Is mandatory to export/import database from 3.0 to 3.1?

### DISCUSSION THREAD

**Discussion by johny65** [orientechnologies/orientdb#9481] [Q&A] [answered]
Having a database working in versión 3.0.x, is mandatory to do the export and then import in 3.1 version? The only official advice I could find is in the manual:

> Between MINOR and MAJOR versions, you may need to export and re-import the database.

But it says you "may need". How I'm sure if I need or not to do the process? For now, opening a 3.0 database directly with 3.1 is working fine, but how can I be sure?

**Comment by andrii0lomakin** [comment, marked as answer]
Hi @johny65 .  "May need" in this context means, that if you do not migrate the database using export/import you will end up on working on old versions of data structures. By data structures, I mean clusters for example. At some cases that means that performance and durability will be lower than you expect from a newer version.

**Comment by johny65** [comment]
Ok, thanks for the answer @laa.
